package/CHANGELOG

```diff
@@ -13,6 +13,13 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
+03/18/22 shudipto-amin
```

Comment:
Unfortunately this is not how the changelog should be updated.

1. Update your branch so you have the latest version of the changelog
2. Add yourself at the current line 23 (on the line starting with my username)
3. Put fixes as a bullet point under Fixes (currently at line 28).

Reply:
Thanks for your instructions! I had some confusion about this part, which is clearer now. I updated the CHANGELOG accordingly, let me know if you have other corrections/suggestions.